src/services/formatting/formatting.ts

```diff
@@ -652,6 +649,11 @@ namespace ts.formatting {
                 if (tokenInfo.token.end > node.end) {
                     break;
                 }
+                if (node.kind === SyntaxKind.JsxText) {
+                    // Intentation rules for jsx text are handled by `indentMultilineCommentOrJsxText` inside `processChildNode`; just fastforward past it here

```

Comment:
Notably, the formatting scanner _actually_ doesn't handle scanning JSX correctly, like, at all; so the only thing that even makes sense is to fastforward over it - otherwise we do silly things like interpret words within the jsxtext as identifiers.